spec/requests/organisations_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'rails_helper'
+
+RSpec.describe "/organisations", type: :request do
+  fixtures :all
+
+  let(:valid_attributes) do
+    { name: 'Foo'  }
+  end
+
+  let(:invalid_attributes) do
+    { name: nil }
+  end
+
+  let(:user) { users :admin }
+
+  before { sign_in user }
```

Comment:
`sign_in` is a method added via the code in the devise support file. It easily allows you to test a page that is restricted via authentication.